site_search.gemspec

```diff
@@ -1,36 +1,30 @@
 
-lib = File.expand_path("../lib", __FILE__)
+lib = File.expand_path('lib', __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require "site_search/version"
+require 'site_search/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "site_search"
+  spec.name          = 'site_search'
   spec.version       = SiteSearch::VERSION
   spec.authors       = ["Tomas D'Stefano"]
-  spec.email         = ["tomas_stefano@successoft.com"]
+  spec.email         = ['development.team@moneyadviceservice.org.uk']
+  spec.summary       = %(Site search is a library which enable to connect
+                       to many services to do the site search.)
+  spec.description = %(Site search is a library which enable to connect
+                       to many services to do the site search.)
```

Comment:
since the description is the same as the summary, should it just be deleted?

Reply:
The gemspec requires a description :\